numpy/core/src/umath/override.c

```diff
@@ -548,15 +554,13 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
             goto fail;
         }
 
-        /* Access the override */
-        array_ufunc = PyObject_GetAttrString(override_obj,
-                                             "__array_ufunc__");
```

Comment:
Again, why look it up a third time? Especially since here we're using a different lookup rule to the previous time